challenge-04/README.md

```diff
@@ -29,53 +47,90 @@ seguintes propriedades (os valores devem ser do tipo mostrado abaixo):
 - `assentos` - Number - cinco por padrão
 - `quantidadePessoas` - Number - zero por padrão
 */
-?
+var carro = {
+	marca : 'Fiat',
+	modelo : 'Uno',
+	placa : 'SON-1215',
+	ano : 2015,
+	cor : 'Preto',
+	quantasPortas : '4',
+	assentos : 5,
+	quantidadePessoas: 0
+
+};
 
 /*
 Crie um método chamado `mudarCor` que mude a cor do carro conforme a cor
 passado por parâmetro.
 */
-?
+carro.mudarCor = function(mudancaCor){
+	carro.cor = mudancaCor; 
+};
 
 /*
 Crie um método chamado `obterCor`, que retorne a cor do carro.
 */
-?
+carro.obterCor = function(){
+	return carro.cor;
+};
 
 /*
 Crie um método chamado `obterModelo` que retorne o modelo do carro.
 */
-?
+carro.obterModelo = function(){
+	return carro.modelo;
+};
 
 /*
 Crie um método chamado `obterMarca` que retorne a marca do carro.
 */
-?
+carro.obterMarca = function(){
+	return carro.marca;
+};
 
 /*
 Crie um método chamado `obterMarcaModelo`, que retorne:
 "Esse carro é um [MARCA] [MODELO]"
 Para retornar os valores de marca e modelo, utilize os métodos criados.
 */
-?
-
+carro.obterMarcaModelo = function(){
+	return 'Esse carro é um ' + carro.obterMarca() + ' ' + carro.obterModelo();
+}
 /*
 Crie um método que irá adicionar pessoas no carro. Esse método terá as
 seguintes características:
-- Ele deverá receber por parâmetro o número de pessoas entrarão no carro. Esse
+- Ele deverá receber por parâmetro o número de pessoas que entrarão no carro. Esse
 número não precisa encher o carro, você poderá acrescentar as pessoas aos
 poucos.
 - O método deve retornar a frase: "Já temos [X] pessoas no carro!"
 - Se o carro já estiver cheio, com todos os assentos já preenchidos, o método
 deve retornar a frase: "O carro já está lotado!"
+
 - Se ainda houverem lugares no carro, mas a quantidade de pessoas passadas por
 parâmetro for ultrapassar o limite de assentos do carro, então você deve
 mostrar quantos assentos ainda podem ser ocupados, com a frase:
 "Só cabem mais [QUANTIDADE_DE_PESSOAS_QUE_CABEM] pessoas!"
+
 - Se couber somente mais uma pessoa, mostrar a palavra "pessoa" no retorno
 citado acima, no lugar de "pessoas".
 */
-?
+carro.adicionarPessoas = function(numeroPessoas){
+	var total = carro.quantidadePessoas + numeroPessoas;
+
+	if(carro.quantidadePessoas === carro.assentos && total >= carro.assentos ){
+		return 'O carro já está lotado!';
+	}
+	
+	if( total > carro.assentos ){
+		var quantidadeCabem = carro.assentos - carro.quantidadePessoas;
+		var quantidadePessoas = quantidadeCabem === 1 ? ' pessoa' : 'pessoas';
+		return 'Só cabem mais '+ quantidadeCabem + quantidadePessoas + ' !';
+	}
+
+	carro.quantidadePessoas += numeroPessoas;
```

Comment:
@andersonweb não entendi sua pergunta.
Vou tentar explicar o que entendi, se não for isso, me avise.
Quando você usa o operador `+=`, você está fazendo duas coisas:
- Você soma todo o valor da variável que está recebendo o valor, e faz essa variável receber o valor atualizado.

Exemplo:

Isso:
```js
var numero = 1;
var outroNumero = 2;
numero = numero + outroNumero;
numero; // agora "numero" vale "3"
```

É o mesmo que fazer:

```js
var numero = 1;
var outroNumero = 2;
numero += outroNumero;
numero // Aqui o número também vale 3. A diferença é que essa é a forma abreviada de atribuir e somar valores ao mesmo tempo :)
```